vividus-extension-web-app/src/main/java/org/vividus/ui/web/storage/IWebStorage.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.ui.web.storage;
+
+import java.util.Set;
+
+public interface IWebStorage
```

Comment:
we avoid old-fashionned `I*` notation for interfaces, instead: `WebStorage` interface and `SeleniumWebStorage` and `PlaywrightWebStorage` classes